addons/gdshell/docs/en/commands/index.md

```diff
@@ -0,0 +1,33 @@
+<a href="https://github.com/Kubulambula/Godot-GDShell">
+  <img src="https://github.com/Kubulambula/Godot-GDShell/blob/main/addons/gdshell/docs/assets/logo.png" align="left" width="80" height="80">
+</a>
+
+# GDShell Command List
+
+All commands are subject to change. This article is simply a list of commands, each command has its own article explaining syntax, flags, and usage. 
+
+<!--
+  - Your README.md says you are trying to mimic the feel of BASH, so
+  - I am trying to make them in line with that, but feel free
+  - to change any you feel could be better!
+  -->
+
+Command arguments in brackets `()` are optional, arguments in `{}` are sometimes required, in `<>` are always required. This article does not exaustively list the arguments, only the basic ones. 
```

Comment:
```suggestion
```

If you mean the bracket types for different argument types, I totally agree with that. However I think that it should be in each individual command's docs page.